src/styles/blocks/navigation.scss

```diff
@@ -0,0 +1,43 @@
+.nav {
```

Comment:
it would be easier to find styles if your files would have the same name as block, for example nav.scss instead of navigation.scss